addons/hr_recruitment_integration_monster/data/monster_salary_time_unit_data.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="monster_salary_time_unit_hours" model="monster.salary.time.unit">
```

Comment:
Is this model really necessary ? A hardcoded mapping between ``salary_time_unit`` and the monster_id is sufficient IMO.

The model should be totally removed.

Reply:
nope I removed it